components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/triggerapi_test.go

```diff
@@ -252,6 +270,39 @@ func TestRunCheck(t *testing.T) {
 		require.Equal(t, result[0].Status, "SUCCESS")
 	})
 
+	t.Run("System Resource Check - PostgreSQL, OpenSearch", func(t *testing.T) {
+		// Create a dummy server
+		server, host, port := createDummyServer()
+		defer server.Close()
+
+		// Test data
+		config := models.Config{
+			Hardware: models.Hardware{
+				PostgresqlNodeCount: 1,
+				PostgresqlNodeIps:   []string{host},
+				OpenSearchNodeCount: 1,
+				OpenSearchNodeIps:   []string{host},
+			},
+		}
+		log := logger.NewLogrusStandardLogger()
+
+		path := constants.SYSTEM_RESOURCE_CHECK_API_PATH
+		depState := "your_deployment_state"
+
+		// Call the function being tested
+		result := RunCheck(config, log, port, path, depState)
+		fmt.Printf("OS PG result: %+v\n", result)
+		require.Equal(t, 2, len(result))
+		require.NotNil(t, result)
+		require.Nil(t, result[0].Error)
+		require.Len(t, result[0].Result.Checks, 2)
+		require.Equal(t, result[0].Status, "SUCCESS")
+		require.Nil(t, result[1].Error)
+		require.Len(t, result[1].Result.Checks, 2)
+		require.Equal(t, result[1].Status, "SUCCESS")
```

Comment:
please add more assert statements to show the result from both the pg and os response